papyrus/protocol.py

```diff
@@ -79,7 +80,7 @@ def create_geom_filter(request, mapped_class, geom_attr):
     box = request.params.get('bbox')
     shape = None
     if box is not None:
-        box = map(float, box.split(','))
+        box = [float(x) for x in box.split(',')]
```

Comment:
In python 3, `map` doesn't return a list but a map object. The problem is that you cannot do `box[0]` on a map object so the code fails with:
```
Traceback (most recent call last):
  File "/home/jenselme/projects/papyrus/papyrus/tests/test_protocol.py", line 94, in test_box_filter_with_epsg
    filter = create_geom_filter(request, MappedClass, "geom")
  File "/home/jenselme/projects/papyrus/papyrus/protocol.py", line 85, in create_geom_filter
    shape = Polygon(((box[0], box[1]), (box[0], box[3]),
TypeError: 'map' object is not subscriptable
```

Hence the switch to list comprehension.